quupe-RN/js/screens/MyItems/MyItems.js

```diff
@@ -1,27 +1,37 @@
 import React from 'react';
```

Comment:
Can you please give me a brief on what is MyItems screen? Thank you

Reply:
LoggedInUserItems*